index.html

```diff
@@ -115,37 +115,6 @@ <h2 class="row-title">About Shropshire Devs</h2>
           <h2 class="row-title">Upcoming Speakers</h2>
 
           <div class="col-md-4 col-sm-6 feature">
-            <img src="img/speaker-mike-taulty.jpg" class="speaker-img">
```

Comment:
Sorry dude, but we could do with adding these outgoing speakers to our previous talks ...